shell/components/Cron/CronExpressionEditor.vue

```diff
@@ -0,0 +1,305 @@
+<script setup lang="ts">
```

Comment:
For new components that will be used in extensions i think we have a pattern of adding them to `pkg/rancher-components`